src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -72,7 +72,8 @@ class CliArgumentType(object):
 
     def __init__(self, overrides=None, **kwargs):
         if isinstance(overrides, str):
-            raise ValueError("Overrides has to be a CliArgumentType (cannot be a string)")
+            raise ValueError(
+                "Overrides has to be a CliArgumentType (cannot be a string)")
```

Comment:
Why are all these changes needed?
Only change in this file should have been to `BLACKLISTED_MODS` I believe.